help/self-hosted-billing.md

```diff
@@ -3,181 +3,202 @@
 This page describes how to manage your self-hosted plan, and answers some common
 questions about plans and billing for self-hosted organizations. Please refer to
 [Self-hosted Zulip plans and pricing](https://zulip.com/plans/#self-hosted) for plan
-details.  If you have any questions not answered here, please don't hesitate to
+details.
+
+The topics covered include:
+
+* [Logging in to manage billing](#log-in-to-billing-management)
+* [Upgrading to a paid plan](#upgrade-to-a-paid-plan)
+* [Managing billing](#manage-billing)
+* [Canceling a paid plan](#cancel-paid-plan)
+* [Applying for a free Community plan](#apply-for-community-plan)
+* [Applying for a paid plan discount](#apply-for-a-paid-plan-discount)
+* [Manual license management](#manually-update-number-of-licenses)
```

Comment:
I originally added this list so that it was easy to navigate between the "log in" instructions block and all of the other instruction blocks on the page.

Now, for example, if you are reading the "apply for Community plan" instructions, click the link for the "log" in instructions, then the user has to use the back button on the browser to navigate back to the application instructions.

I'm fine if folks disagree and prefer this placement, but I thought I would explain why I originally placed this list after the "log in" instructions and not here.

Reply:
I added a sentence below the login instructions, so now we have the links there too:

![Screenshot 2024-06-12 at 12 34 37@2x](https://github.com/zulip/zulip/assets/2090066/05f0a256-ab0d-4d61-9e29-b29c1891dfa3)

I think that gets us the best of both worlds.